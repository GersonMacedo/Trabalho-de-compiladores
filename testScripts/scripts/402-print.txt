! -s RUN --disable-logs printer
! should print correctly
program p ;
begin
    print 5;
    print 42;
    print false;
    print true;
end .