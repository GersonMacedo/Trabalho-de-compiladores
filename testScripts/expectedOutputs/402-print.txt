
Performing build generation and running at testScripts/scripts/402-print.txt
No errors found
Building the program

      LOADL     5
      CALL      putint
      CALL      puteol
      LOADL     42
      CALL      putint
      CALL      puteol
      LOADL     0
      CALL      putint
      CALL      puteol
      LOADL     1
      CALL      putint
      CALL      puteol
      HALT
Builder::Writing object to obj.tam
********** TAM Interpreter (Java Version 2.1) **********
5
42
0
1

Program has halted normally.
